release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -243,6 +243,18 @@ no operators found with name <name>.<version> in channel <name> of package <name
 
 === Operator development
 
+[id="ocp-4.8-sandboxed-containers"]
+=== {sandboxed-containers-first}
+
+[id="ocp-4.8-sandboxed-containers-tp"]
+==== {sandboxed-containers-first} support on {product-title} (Technology Preview)
+
+{sandboxed-containers-first} 1.0.0 Technology Preview release introduces built-in support for running Kata Containers as an additional runtime. {sandboxed-containers-first} enables users to choose Kata Containers as an additional runtime to provide additional isolation for their workloads. The {sandboxed-containers-operator} enables users to perform tasks such as installation, removal, updates, and status monitoring of Kata Containers.
```

Comment:
I watched this in preview mode, it displays as "OpenShift sandboxed containers 1.0.0 Technology Preview". What confuses me is that the only non-capitalized part here is the "most essential" part of the product name.

Could this be changed to show as "OpenShift Sandboxed Containers 1.0.0 Technology Preview"? Or alternatively, show the name in some different style?

Reply:
@c3d the product name is OpenShift sandboxed containers

Reply:
@zanetworker I understand that this is how we currently capitalize it. I still feel the need to point out that the following looks and reads wrong:

_OpenShift sandboxed containers 1.0.0 Technology Preview release introduces built-in support for running Kata Containers as an additional runtime. OpenShift sandboxed containers enables users to choose Kata Containers as an additional runtime to provide additional isolation for their workloads. The OpenShift sandboxed containers Operator enables users to perform tasks such as installation, removal, updates, and status monitoring of Kata Containers._

Maybe simply using a different font, e.g.:

_OpenShift sandboxed containers 1.0.0 Technology Preview release_ introduces built-in support for running _Kata Containers_ as an additional runtime. _OpenShift sandboxed containers_ enables users to choose _Kata Containers_ as an additional runtime to provide additional isolation for their workloads. The _OpenShift sandboxed containers Operator_ enables users to perform tasks such as installation, removal, updates, and status monitoring of _Kata Containers_.

Notice how the Parts in italicS Are totally Inconsistent in Their capitAlization.

Reply:
@c3d Discussions about this has happened previously. The approved product name is OpenShift sandboxed containers. Per our guidelines we have to capitalize our Operators. However, the approved product name is OpenShift sandboxed containers and OpenShift sandboxed containers Operator. Kata Containers is what I have found in their documentation in terms. It looks weird definitely, but this is what has been approved.

Reply:
Which documentation are you referring to @sagidlow. I think Christoph's suggestion is trying to distinguish usage visually using italics as an example.

Reply:
@zanetworker I suspect that @sagidlow is referring to how Kata Containers is spelled on their web site, e.g. https://katacontainers.io. It is rather consistently spelled with two capitals.

Frankly, even if that's something that needs to be revisited at the product management level, I think it's worth bringing up the issue that the current capitalisation makes documentation unreadable. In particular if we start capitalizing "operator"! It seriously takes me three or four back and forth scans in a sentence to simply be able to parse "OpenShift sandboxed containers 1.0.0 Technology Preview release introduces built-in support". To me, this is bad enough to turn into a UX experience at this point.

Different fonts could definitely help. Or the use of well-defined acronyms. If those are not options, then we need to pay attention to what the phrasing looks like in the end, something that may be a bit obscured to the writers because of the use of macros. For example 

_The release of the Technical Preview for OpenShift sandboxed containers 1.0.0 introduces built-in support for running Kata Containers as an additional runtime_

Reply:
@zanetworker @c3d - With our current guidelines we cannot use italics in this manner. We also cannot implement different fonts, and so on. 

We can talk about changing the name of the Operator or the product itself by using proper capitalization. But unfortunately, unless we can fix the product name. As @c3d this is def. a UX experience issue and if we can change OpenShift sandboxed containers to OpenShift Sandboxed Containers that would make more sense and more align with how our products are titled.